modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/paths/ProcessBuilderKaleoDesignerReact.path

```diff
@@ -89,6 +89,16 @@
 	<td>//div[@class='autofit-section' and contains(.,'${key_translationOption}')]//..//..//span[@class="label label-${key_translationStatus}"]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>TRANSLATION_COUNTRY_LABEL_TEXT</td>
+	<td>//*[@role='menuitem'][@class='dropdown-item'][.//div[contains(.,'${key_translationOption}')]] //*[contains(@class,'label-item')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>TRANSLATION_LANGUAGE_MENU_ITEM</td>
+	<td>//*[@role='menuitem'][@class='dropdown-item'][.//div[contains(.,'${key_translationOption}')]]</td>
+	<td></td>
+</tr>
```

Comment:
Hey @patriciajeanperez ! I was unable to use `PageEditor.changeLocale`, since this macro uses twice the variable `locale`.  The first path works because it uses the html text value with an `_`, but the second path doesn't work since it uses `lexicon-icon-${key_currentLocale}`, at at this point need to be with an `-`. And the variable being passed from the Testcase uses an `_`.
I can add a `StringUtil.replace` to changeLocale between the functions to make it work, but I'm unsure if this is the best solution.

Reply:
But nevertheless, I was able to reuse some paths from Translation and I amended to the commits.

Reply:
Nice, thanks Luiz!